hardpy/pytest_hardpy/db/base_store.py

```diff
@@ -177,3 +182,7 @@ def _init_doc(self) -> dict:
         }
 
         return doc
+
+
+# Backward compatibility alias
+BaseStore = CouchDBStore
```

Comment:
This is no longer BaseStore and also the file name should reflect, that it is now CouchDB related